src/content/reference/cluster-upgrades/_index.md

```diff
@@ -41,44 +41,62 @@ Among the third party components building a tenant cluster stack are
 as well as many operators and controllers created and maintained by Giant Swarm.
 
 All of the items in the list above are released independent of each other by their vendors.
-At Giant Swarm we bundle specific versions of these components of the tenant cluster stack into a **tenant cluster release**. A tenant cluster release is specific for a provider (AWS, Azure, or KVM) and identified by a version number. To learn more about tenant cluster releases and our versioning, check the [tenant cluster releases reference](/reference/tenant-cluster-release-versions/).
+At Giant Swarm we bundle specific versions of these components of the workload cluster stack into a **workload cluster release**. A workload cluster release is specific for a provider (AWS, Azure, or KVM) and identified by a version number. To learn more about workload cluster releases and our versioning, check the [workload cluster releases reference](/reference/tenant-cluster-release-versions/).
 
-Once deployed, the tenant cluster stack is **immutable**.
+Once deployed, the workload cluster stack is **immutable**.
 All components are deployed based on images, either of virtual machines or of Docker containers.
 No changes are ever made to components at runtime.
 This ensures that the stack running in your environment is the exact stack we have tested before.
 
-As a consequence, the only way to change the stack is to perform an upgrade, to switch to a new tenant cluster release.
+As a consequence, the only way to change the stack is to perform an upgrade, to switch to a new workload cluster release.
 
 ### Upgrade semantics {#semantics}
 
-According to our tenant cluster release versioning, three different levels of upgrades can occur:
+According to our workload cluster release versioning, three different levels of upgrades can occur:
 
 - *Patch upgrade*: Only the patch version number is increased.
 
 - *Minor upgrade*: The minor version number is increased, while the Major version number stays the same.
```

Comment:
There is an inconsistency between capitalization of `minor` and `Major`

Reply:
done, by consistency I put it in lowercase if does not start the phrase